packages/babel-plugin/index.js

```diff
@@ -0,0 +1,18 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const tsNode = require('ts-node');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const tsPaths = require('tsconfig-paths');
+
+tsNode.register({
```

Comment:
Since the babel plugin is the first to be picked up its setting up tsnode + tsconfig paths. This is used both by storybook and by jest.

There could be a better way to set this up.